resources/views/admin.blade.php

```diff
@@ -197,30 +201,18 @@
                 });
 
                 //hideLongText
-                function ellipsis(){
-                    truncateColumns('column-company' , 26)
-                    truncateColumns('column-branch' , 15)
-                    truncateColumns('column-name' , 26)
-                    truncateColumns('column-position' , 20)
-                    truncateColumns('column-address' , 50)
-                }
-
-                function truncateColumns(columnName , charLimit){
+                function hideLongText(columnName , charLength){
```

Comment:
I noticed you call the `hideLongText` parent function where it has a child function which is looping over all the cells in specified columns.  which should be avoided, consider if we nested multiple tables. 

Please explain this method, and mention is there any alternatives available.

Reply:
yes i can understand didn't give much thought at first...i tried so many options and methods..it seems liked nothing can be done without columnDef...so i implemented it using js...

hideLongText(columnName , charLength) is a simple function.....
it take all the rows of a column then it compares the text length f the content in the row with a given number. If its more than  that value it substring and creates a tool tip or else only tool tip...changes are done to the Dom element it self 

While i was searching i cam across the a best method to implement this feature using a extension built for data table but couldn't find a way to implement it without the target id...if its possible give me a hint please

https://datatables.net/plug-ins/dataRender/ellipsis


@Raza9798 #60

Reply:
refactored the by referencing the example please review again #60  @Raza9798